tests/Sitemap/SitemapFactoryTest.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+namespace Test\AppBundle\Sitemap;
+
+use AppBundle\Entity\Article;
+use AppBundle\Entity\ArticleCategory;
+use AppBundle\Entity\OrderArticle;
+use AppBundle\Repository\ArticleCategoryRepository;
+use AppBundle\Repository\ArticleRepository;
+use AppBundle\Repository\OrderArticleRepository;
+use AppBundle\Sitemap\SitemapFactory;
+use Doctrine\Common\Persistence\ObjectManager;
+use Psr\Cache\CacheItemInterface;
+use Psr\Cache\CacheItemPoolInterface;
+use Symfony\Bundle\FrameworkBundle\Tests\TestCase;
+use Symfony\Component\Routing\RouterInterface;
+use Tackk\Cartographer\Sitemap;
+
+class SitemapFactoryTest extends TestCase
+{
+    private $objectManager;
+    private $routerInterface;
+    private $cacheItemPoolInterface;
+    private $article;
+    private $category;
+
+    public function testAddArticle()
```

Comment:
missing return type on some methods of this class

Reply:
In this projets, there is no return type on testXXX method.